spec/models/services/facebook_spec.rb

```diff
@@ -54,7 +82,18 @@
 
     it "should include post url in message with photos" do
       post_params = @service.create_post_params(@status_message)
-      expect(post_params[:message]).to include 'http'
+      expect(post_params[:message]).to include '(via http:'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.